pandas/tests/computation/test_eval.py

```diff
@@ -1890,6 +1891,26 @@ def test_negate_lt_eq_le(engine, parser):
         tm.assert_frame_equal(result, expected)
 
 
+@pytest.mark.parametrize(
+    "column",
+    DEFAULT_GLOBALS.keys(),
+)
+def test_eval_no_support_column_name(request, column):
+    if column in ["True", "False", "inf", "Inf"]:
+        request.node.add_marker(
+            pytest.mark.xfail(
+                raises=KeyError,
+                reason=f"DataFrame eval not supported with {column}",
```

Comment:
Could you add the Github issue to this `reason` that notes that True, False, etc are not supported?

Reply:
Have add github issue https://github.com/pandas-dev/pandas/issues/47859